docs/usage/rust.md

```diff
@@ -11,14 +11,19 @@ Renovate supports upgrading dependencies in `Cargo.toml` files and their accompa
 
 1. Renovate searches in each repository for any `Cargo.toml` files
 1. Renovate extracts existing dependencies from `[dependencies]`, `[dev-dependencies]`, `[build-dependencies]` and `[workspace.dependencies]`
-1. Renovate tries to find and parse a `.cargo/config.toml` file to discover index URLs for private registries
+1. Renovate looks up Cargo configuration to discover index URLs for private registries.
```

Comment:
```suggestion
1. Renovate looks up Cargo configuration to find index URLs for private registries
```

We don't use punctuation at the end of a list item. I like `find` better than `discover`.